bin/eds-import-from-figma.js

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env node
+(async function () {
+  const fs = require('fs');
+  const jsonfile = require('jsonfile');
+  const yargs = require('yargs/yargs');
+
+  // eslint-disable-next-line import/extensions
+  const { hideBin } = require('yargs/helpers');
+  const { prompt } = require('enquirer');
+  const at = require('lodash/at');
+  const set = require('lodash/set');
+
+  const { default: ora } = await import('ora');
+  const chalk = require('chalk');
+
+  // read in the config from config file, package json "eds", etc.
+  const { getConfig } = require('./_util');
+  const config = await getConfig();
+
+  // Set up the command structure to output help if the file to load is not specified
+  const args = yargs(hideBin(process.argv))
+    .command('$0 <file>', 'Figma variable file to import from', (yargs) => {
+      yargs.positional('file', {
+        describe: 'JSON tokens file emitted from the plugin',
+        type: 'string',
+      });
+    })
+    .option('verbose', {
+      describe: 'Print additional details for debugging purposes',
+      type: 'boolean',
+    })
+    .option('dry-run', {
+      describe: 'Run the script without writing any changes',
+      type: 'boolean',
+    }).argv;
```

Comment:
![Screenshot 2024-02-09 at 17 06 13](https://github.com/chanzuckerberg/edu-design-system/assets/3056447/8421a150-5f80-4696-aaa1-71b525a1b810)

Reply:
Is the first line of the print statement accurate?

If someone is running this script would they use the command `eds-import-from-figma <file>` and not `eds-import-from-figma.js <file>`

Reply:
`yargs` will figure out the command to run dynamically, based on how the command was triggered. It will be a different command once this is packaged, BUT a bit unclear what will show on screen